antarest/core/config.py

```
@@ -417,38 +435,36 @@ class LauncherConfig:
     """
 
     default: str = "local"
-    local: Optional[LocalConfig] = None
-    slurm: Optional[SlurmConfig] = None
+    configs: Optional[List[LocalConfig | SlurmConfig]] = None
     batch_size: int = 9999
 
     @classmethod
     def from_dict(cls, data: JSON) -> "LauncherConfig":
+        launchers: List[LocalConfig | SlurmConfig] = []
         defaults = cls()
         default = data.get("default", cls.default)
-        local = LocalConfig.from_dict(data["local"]) if "local" in data else defaults.local
-        slurm = SlurmConfig.from_dict(data["slurm"]) if "slurm" in data else defaults.slurm
         batch_size = data.get("batch_size", defaults.batch_size)
+        for launcher in data["launchers"]:
+            if launcher["type"] == Launcher.LOCAL:
+                launchers.append(LocalConfig.from_dict(launcher))
+            elif launcher["type"] == Launcher.SLURM:
+                launchers.append(SlurmConfig.from_dict(launcher))
+            else:
+                continue
```

we should raise an error here instead.
Note that you may use a `match` instead of if/else